turbo.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "https://turbo.build/schema.json",
+  "globalDependencies": [".env"],
```

Comment:
We probably want to instead explicitly list all of the env vars here 👍

Reply:
Pushed an update for this: https://turbo.build/repo/docs/reference/configuration#globalenv